sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Release History
 
-## 1.3.1 (Unreleased)
+## 1.4.0b1 (Unreleased)
+- Added `AzureCliCredential`, which authenticates with the identity logged in
+to the Azure CLI. This credential is part of `DefaultAzureCredential` by
```

Comment:
Please sync with rest of lang devs to ensure we are on the same page across langs on whether or not DAC has CLICred by default.

I highly prefer it to be enabled by default so developer doesn't have to do anything to configure it.